ga4gh/backend.py

```diff
@@ -6,9 +6,18 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import ga4gh.datamodel as datamodel
 import ga4gh.exceptions as exceptions
 import ga4gh.protocol as protocol
+import ga4gh.registry as registry
+
+# We must import these modules before queries are run, or we won't be
+# able to load the polymorphic types at run time. These should
+# probably be imported in the frontend so that we are more
+# configurable and open the possibility of a plugin architecture.
+import ga4gh.datasource.simulator  # noqa
```

Comment:
We're not actually referring to this package anywhere in the module, so pep8 gets upset. If we leave them out, SQL Alchemy breaks. Pr